python/py-genshi/Portfile

```diff
@@ -1,17 +1,18 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
```

Comment:
please keep getting the file from PyPI, which is the preferred source for Python packages. 

Just set `python.rootname Genshi` below the name and you can remove the `livecheck`, `master_sites`, `distname` stuff.